app/src/main/java/org/mozilla/fenix/settings/AccessibilityFragment.kt

```diff
@@ -25,6 +26,23 @@ class AccessibilityFragment : PreferenceFragmentCompat() {
         (activity as AppCompatActivity).title = getString(R.string.preferences_accessibility)
         (activity as AppCompatActivity).supportActionBar?.show()
 
+        val forceZoomPreference = findPreference<SwitchPreference>(
+            getPreferenceKey(R.string.pref_key_accessibility_force_enable_zoom)
+        )
+        forceZoomPreference?.isVisible = FeatureFlags.forceZoomPreference
```

Comment:
nit could probably be a bit cleaner all within a `?.apply` after finding preference?